File: java/lv1/CokeIssues.java
package lv1;

public class CokeIssues {
    public int solution(int a, int b, int n) {
        int received;
        int glass = n;
        int coke = 0;
        while (glass >= a) {
            received = (glass / a) * b;
            coke += received;
            glass = received + glass % a;
        }
        return coke;
    }
}

File: kotlin/lv1/CokeIssues.kt
package lv1

class CokeIssues {
    fun solution(a: Int, b: Int, n: Int): Int {
        var coke = 0;
        var received = 0;
        var glass = n;
        while (glass >= a) {
            received = (glass / a) * b;
            glass = received + (glass % a);
            coke += received;
        }
        return coke;
    }
}